network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/news_you_can_use.html

```diff
@@ -30,9 +30,11 @@ <h2 class="h4-heading mb-4">{% trans "News you can use" %}</h2>
             {% image first.get_meta_image fill-1000x500 %}
           </div>
           <div class="feature-content align-self-center">
-            {% with category=first.category.first %}
-              <a class="h6-heading d-block mb-2" href="/blog/category/{{ category.slug }}">{{ category }}</a>
-            {% endwith %}
+            {% if item.blog.category.count %}
+              {% with category=first.category.first %}
+                <a class="h6-heading d-block mb-2" href="/blog/category/{{ category.slug }}">{{ category }}</a>
+              {% endwith %}
+            {% endif %}
```

Comment:
As per the issue, this still needs _something_ in place of that link to ensure that the headings line up, so at the very least this needs an `{% else %}<span>&nbsp;</span>{% endif %}` or something.

Reply:
Ahh right, fair enough. Ill add in an nbsp 👍

Reply:
Desktop preview:
![image](https://user-images.githubusercontent.com/4743971/94827683-f6dc3780-03c5-11eb-95a5-a370b3270bae.png)

Mobile preview (I'm hiding the empty category on mobile so there isn't an awkward empty space):
![image](https://user-images.githubusercontent.com/4743971/94827765-0ce9f800-03c6-11eb-8e8d-c5296c4d7ee4.png)